postgres/test/test_postgres.py

```diff
@@ -110,6 +110,7 @@ def test_check(self):
         host = 'localhost'
         port = 15432
         dbname = 'datadog_test'
+        instance_tag = 'postgresinstance: additionalTag'
```

Comment:
Space aren't allowed in tags, even if the test works it would be better to remove it.